### DISCUSSION TITLE
Retweetlemede sorunmu var?

### DISCUSSION THREAD

**Discussion by azerbaijann** [IBAX-io/go-ibax#400] [Bug Submission]
Retwetlemede sorunmu var?

**Comment by plankton4236** [comment]
BENCE DE PROBLEM VAR SONRA DENEYİNİZ DİYOR

**Comment by Voltrader23** [comment]
evet sorun var

**Comment by ozcanizm** [comment]
bende sıkıntı yaşıyorum

**Comment by mackcarelock** [comment]
Hep problem var diyor anlamadım. Kabul etmiyor retwitleri , puanlamıyor.

Bir de benim puan bölümünde dashboard da puanlarım doğru görünüyor fakat private key ile wallet a giriş yaptığımda puanlarim sıfırlanıyor. Bu neden oluyor

**Comment by gokhan85** [comment]
reetwitte sorun yok.benim gorevlerim tamamlandı lütfen yarin tekrar gel diyor
